diff --git a/node_modules/design-react-kit/dist/Upload/UploadDragNdrop.js b/node_modules/design-react-kit/dist/Upload/UploadDragNdrop.js
index ae00e23..ada68f7 100644
--- a/node_modules/design-react-kit/dist/Upload/UploadDragNdrop.js
+++ b/node_modules/design-react-kit/dist/Upload/UploadDragNdrop.js
@@ -1,7 +1,8 @@
 import classNames from 'classnames';
 import React, { useState } from 'react';
 import { Icon } from '../Icon/Icon';
-import DragandDropIcon from '../../assets/images/upload-drag-drop-icon.svg';
+// removed icon that was making the build file, remove thisn patch when fixed upstream
+// https://github.com/italia/design-react-kit/issues/1192
 export const UploadDragNdrop = ({ files, setFiles }) => {
     const [dragOverClass, setDragOverClass] = useState(false), dragOverClasses = classNames('upload-dragdrop', {
         dragover: dragOverClass,
@@ -34,7 +35,6 @@ export const UploadDragNdrop = ({ files, setFiles }) => {
     };
     return (React.createElement("div", { className: dragOverClasses },
         React.createElement("div", { className: 'upload-dragdrop-image', onDrop: handleDrop, onDrag: handleDrag, onDragStart: handleDrag, onDragEnd: handleDrag, onDragExit: handleDrag, onDragOver: handleDragOver, onDragEnter: handleDrag, onDragLeave: handleDragLeave },
-            React.createElement("img", { src: DragandDropIcon, alt: 'descrizione immagine', "aria-hidden": 'true' }),
             files.length > 0 && (React.createElement("div", { className: 'upload-dragdrop-success' },
                 React.createElement(Icon, { icon: 'it-check', size: 'xs' })))),
         React.createElement("div", { className: 'upload-dragdrop-text' },
